packages/embeds/embed-core/src/embed.ts

```diff
@@ -18,10 +27,20 @@ customElements.define("cal-inline", Inline);
 declare module "*.css";
 type Namespace = string;
 type Config = {
-  origin: string;
+  calOrigin: string;
```

Comment:
Makes the naming for calOrigin consistent. At few places it was being referred as `origin` and at some other places it is `calOrigin`.